### PR TITLE
Trivy ignore helm

### PR DESCRIPTION
**Pull request by jackstine**
# Description

Updating Trivy ignore list because the impact of Helm upgrade is minimal, and does not impact the control and data planes of Gloo. Only glooctl can be impacted on helm values inserted during install.

# Context

Helm requires a much larger upgrade to the k8s/api which is not ready at this moment. Since the impact is minimal because Gloo data and control planes are not impacted, the helm module upgrade can be ignored.

# Checklist:

- [ ] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [ ] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [ ] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [ ] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: projects/gloo/cli/pkg/cmd/install/installer.go [hunk 1]
```diff
@@ -146,6 +146,7 @@ func (i *installer) installFromConfig(installerConfig *InstallerConfig) error {
 
 	// Merge the CLI flag values into the extra values, giving the latter higher precedence.
 	// (The first argument to CoalesceTables has higher priority)
+	// TODO-JAKE installing from the config can cause issue.
 	completeValues := chartutil.CoalesceTables(installerConfig.ExtraValues, cliValues)
 	if installerConfig.Verbose {
 		b, err := json.Marshal(completeValues)
```
Comment (jackstine) [inline projects/gloo/cli/pkg/cmd/install/installer.go:152]: this seems to be the only impact

**Review by sam-heilbron — COMMENTED** [review]
This is awesome! Can you include the steps/commands someone can take to run the trivy scans locally to verify that adding these to the `.trivyignore` works as we expect?

**Comment by jackstine** [comment]
#See Trivy Results
Do the following actions for both `master` and `trivy-ignore-helm`. In the `trivy-ignore-helm` branch you should see no vulnerabilities were found.

##Build the images
I am using the Version `v1.13.1-trivy-test`
`VERSION=v1.13.1-trivy-test ./ci/deploy-to-kind-cluster.sh`

##Run Trivy scans on Images
Run the following script to generate the trivy reports.
```bash
images=(
  /access-logger
  /certgen
  /discovery
  /gloo
  /gloo-envoy-wrapper
  /ingress
  /sds
)

versions=(
    1.13.1
    v1.13.1-trivy-test # this script assumes that all gloo images have been built with this tag
)

registries=(
    quay.io/solo-io
    localhost:5000    # and that all image registries are localhost 5000 as well
)

# for each tag, run a trivy scan on the images
for image in "${images[@]}"; do
  for i in "${!versions[@]}"; do
    fullImage=${registries[$i]}${image}":"${versions[$i]}
    version=${versions[$i]}
    echo "scanning "${fullImage}
    trivy image --severity HIGH,CRITICAL --exit-code 1 --no-progress $fullImage >> trivy_results_$version.txt
  done
done
```

## Verify trivy scans
You should see no vulnerabilites.
```bash
cat trivy_results_1.13.1.txt
cat trivy_results_v1.13.1-trivy-test.txt
```

**Review by sam-heilbron — APPROVED** [review]
Since we're not changing the source code, and just affecting the configuration we provide during our scanning, I ran:
```
VERSION=1.13.1 make scan-version -B
```

and it produced:
```
PATH=/Users/samheilbron/go/src/github.com/solo-io/gloo/_output/.bin:$PATH GO111MODULE=on go run github.com/solo-io/go-utils/securityscanutils/cli scan-version -v \
                -r quay.io/solo-io\
                -t 1.13.1\
                --images gloo,gloo-envoy-wrapper,discovery,ingress,sds,certgen,access-logger,kubectl
{"level":"info","ts":"2022-12-30T13:03:06.641-0700","caller":"commands/scan_version.go:73","msg":"Starting ScanVersion with version=1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:18.149-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 11.507539483s on quay.io/solo-io/gloo:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:18.149-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/gloo:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"debug","ts":"2022-12-30T13:03:23.972-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 5.823046749s on quay.io/solo-io/gloo-envoy-wrapper:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:23.972-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/gloo-envoy-wrapper:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"debug","ts":"2022-12-30T13:03:28.426-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 4.453781399s on quay.io/solo-io/discovery:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:28.426-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/discovery:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"debug","ts":"2022-12-30T13:03:37.650-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 9.224253659s on quay.io/solo-io/ingress:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:37.650-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/ingress:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"debug","ts":"2022-12-30T13:03:41.373-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 3.722363105s on quay.io/solo-io/sds:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:41.373-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/sds:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"debug","ts":"2022-12-30T13:03:45.906-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 4.533491744s on quay.io/solo-io/certgen:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:45.906-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/certgen:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"debug","ts":"2022-12-30T13:03:50.575-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 4.668573619s on quay.io/solo-io/access-logger:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:50.575-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/access-logger:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"debug","ts":"2022-12-30T13:03:54.493-0700","caller":"securityscanutils/trivy_scanner.go:83","msg":"Trivy returned 0 after 3.918325581s on quay.io/solo-io/kubectl:1.13.1"}
{"level":"debug","ts":"2022-12-30T13:03:54.493-0700","caller":"commands/scan_version.go:100","msg":"Scanned Image: quay.io/solo-io/kubectl:1.13.1, ScanCompleted: true, VulnerabilityFound: false, Error: <nil>"}
{"level":"info","ts":"2022-12-30T13:03:54.493-0700","caller":"commands/scan_version.go:31","msg":"No vulnerabilities found"}
```

We could also verify using custom images, with:
```
VERSION=custom make docker -B
```

```
VERSION=custom make scan-version -B
```

**Review by nfuden — APPROVED** [review]
Thanks for the thorough comment. Also glad to see we have a follow up issue for when we have go 1.19 working

**Pull request merged**